### DISCUSSION TITLE
SnackbarStack title not displayed

### DISCUSSION THREAD

**Discussion by dockinkong** [Megabit/Blazorise#3135] [Q&A] [answered]
Title parameter is ignored, null used instead [link to code](https://github.com/Megabit/Blazorise/blob/master/Source/Extensions/Blazorise.Snackbar/SnackbarStack.razor.cs#L121) :

`public Task PushAsync( string message, string title = null, SnackbarColor color = SnackbarColor.None, Action<SnackbarOptions> options = null )
        {
            return PushAsync( (MarkupString)message, null, color, options );
        }`

**Comment by stsrki** [comment, marked as answer]
Goof catch, thanks. I will create an issue.
